book/quick-brick/tvOS/Plugins/intro.md

```diff
@@ -5,4 +5,7 @@
     * [Google Analytics](/quick-brick/tvOS/Plugins/Analytics/GoogleAnalytics/GoogleAnalytics.md)
 * [Player](/quick-brick/tvOS/Plugins/Video/intro.md)
     * [How To Create New Plugin?](/quick-brick/tvOS/Plugins/Video/VideoPluginsHowCreate.md)
-    * [Default Player](/quick-brick/tvOS/Plugins/Video/DefaultPlayer/DefaultPlayer.md)
\ No newline at end of file
+    * [Default Player](/quick-brick/tvOS/Plugins/Video/DefaultPlayer/DefaultPlayer.md)
+* [Player Dependant](/quick-brick/tvOS/Plugins/PlayerDependants/intro.md)
```

Comment:
just a comment across the entire PR
it's Dependent, not Dependant , or Depandant :)
Please search and fix the entire PR accordingly.
INCLUDING file names